caniusepython3/test/test_dependencies.py

```diff
@@ -18,11 +18,19 @@
 from caniusepython3 import dependencies
 from caniusepython3.test import mock, unittest
 
-import io
-
 
 class GraphResolutionTests(unittest.TestCase):
 
+    def test_circular_dependencies(self):
+        reasons = {
+            'A': 'C',
+            'B': 'C',
+            'C': 'A',
```

Comment:
Why not have the dependencies go A -> B, B -> C, C -> A to get real depth of circularity? If you want a dependency that isn't circular as well then C -> D would also work.